lib/_getAa.ts

```diff
@@ -0,0 +1,10 @@
+import { isFunction } from "./utils";
+import AlgoliaAnalytics from "./insights";
+
+export default function getAa(instance: AlgoliaAnalytics) {
```

Comment:
We can use a meaningful name for the function. From the import it's not very clear what it is.

Reply:
https://github.com/algolia/search-insights.js/pull/140/commits/ade891e74b97e69733490d66127fedac246999ca